NOTICE

```diff
@@ -1,5 +1,5 @@
 Splunk SOAR IronNet
-Copyright (c) IronNet, 2020
```

Comment:
This is update so it should be `Copyright (c) IronNet, 2020-2022` in all places

Reply:
done